pkg/cmd/pulumi/config.go

```diff
@@ -39,6 +39,10 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 )
 
+type StackConfigOptions struct {
```

Comment:
I think we can just get rid of this, I don't forsee any options here and applyProjectConfig should _always_ be true for now.